### DISCUSSION TITLE
GridSplitter implementation in MAUI, that works simular to the WPF GridSplitter

### DISCUSSION THREAD

**Discussion by CiRazDeveloper** [CommunityToolkit/Maui#1810] [New Feature Discussions]
With this message I announce the proposal to add the GridSplitter to MAUI that I am currently working on. It will have a very similar worklow, like the WPF GridSplitter. I saw that this feauture is asked by many people and since I recently started the process and was able to add the resizing of the previous and next column, I think that many people will be happy to use it.

https://github.com/CommunityToolkit/Maui/assets/145051936/09e695db-bd61-4091-a62d-eb59b4342126

https://github.com/CommunityToolkit/Maui/assets/145051936/8ffed291-e7ed-4071-843b-fbf33bfa6a7c

**Comment by CliffAgius** [comment]
Did you get this working?

I am closing as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers. 

If you feel this is being closed in error please let us know and we can re-open.
